go.mod

```diff
@@ -300,7 +300,7 @@ replace github.com/hashicorp/go-version => github.com/6543/go-version v1.3.1
 
 replace github.com/shurcooL/vfsgen => github.com/lunny/vfsgen v0.0.0-20220105142115-2c99e1ffdfa0
 
-replace github.com/nektos/act => gitea.com/gitea/act v0.243.4
+replace github.com/nektos/act => gitea.com/sorenisanerd/act v0.246.1-0.20230704003127-0ffa64475fd8
```

Comment:
Please send PR to gitea.com/gitea/act first.

Reply:
I did. https://gitea.com/gitea/act/pulls/73